SpiffWorkflow/bpmn/PythonScriptEngineEnvironment.py

```diff
@@ -0,0 +1,112 @@
+import copy
+
+class BasePythonScriptEngineEnvironment:
```

Comment:
This is the interface I always wanted for SpiffWorkflows Python execution. Really beautiful.